azure/databricks_notebook.py

```
@@ -138,18 +146,162 @@ def inject_job_path(job_config: Dict[str, Any]) -> str:
     return f"{job_config['TARGET_DIRECTORY']}/{target_job_name}"
 
 
-def deploy_requirements_to_databricks(requirements: Dict[str, Any]) -> None:
-    #  Todo: handle requirements through libraries api
-    raise NotImplementedError
+def get_latest_maven_artifact(requirement_name: str) -> str:
+    artifact_parts = requirement_name.split(':')
+    if len(artifact_parts) != 3:
+        raise ValueError(f'Given maven artifact {requirement_name} is invalid, '
+                         f'must be in format GROUP:ARTIFACT:VERSION')
+    group_id = artifact_parts[0]
+    artifact_id = artifact_parts[1]
+
+    url = 'https://repository.sonatype.org/service/local/artifact/maven/redirect'
+    headers = {"content-type": "application/json"}
+    params = {"r": "central-proxy",
+              "g": group_id,
+              "a": artifact_id,
+              "v": "LATEST"}
+    r = call_get(url, headers=headers, params=params, error_message='Error in maven version check')
+    return r.url.split('/')[-2]
```

is there a need to check the url split separately and throw more context aware exception if expected parts are not present?